pkg/kudoctl/cmd/diagnostics/runner_helper.go

```diff
@@ -23,6 +23,24 @@ func diagForInstance(instance string, options *Options, c *kudo.Client, info ver
 		return err
 	}
 
+	deps, err := newDependenciesResources(instance, options, c, s)
+	if err != nil {
+		p.printError(err, DiagDir, "instance")
+		return err
+	}
+
+	for _, dep := range deps {
+		ctx := &processingContext{root: DiagDir, instanceName: dep.instanceObj.Name}
+
+		runner := runnerForInstance(dep, ctx)
+		runner.addObjDump(info, ctx.rootDirectory, "version")
+		runner.addObjDump(s, ctx.rootDirectory, "settings")
```

Comment:
We don't need those two lines here - they just save version and settings into files in the root directory